azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/models/CallCompositeCallStateChangedEvent.java

```diff
@@ -8,14 +8,26 @@
  */
 public final class CallCompositeCallStateChangedEvent {
     private final CallCompositeCallStateCode code;
+    private final String callId;
+    private final int callEndReasonCode;
+    private final int callEndReasonSubCode;
 
     /**
      * Create {@link CallCompositeCallStateChangedEvent} with call state.
      *
      * @param code call state {@link CallCompositeCallStateCode}.
+     * @param callEndReasonCode the call end reason code.
+     * @param callEndReasonSubCode the call end reason sub code.
+     * @param callId the call id.
      */
-    public CallCompositeCallStateChangedEvent(final CallCompositeCallStateCode code) {
+    public CallCompositeCallStateChangedEvent(final CallCompositeCallStateCode code,
```

Comment:
I think it is a breaking change.
We made constructor public by mistake, as there is no intention for Contoso to create instance of `CallCompositeCallStateChangedEvent`. But it is already public an we can't just add parameters.
If ARB decides it is OK to make such breaking change, then I would rather make this constructor internal.